Api testing

harrison-blake opened the pull request:
### What?
 * Testing for facade, service, and poro

### Why?
 * Test to see if functioning correctly

### Testing?
 * happy path testing for facade, service, and poro

Closes #43

Commit:
Add search test for blank input

Commit:
Add spec files for testing facade, service and fixture with initial tests

Commit:
Finish Happy path tests for poro facade and service

Gvieve commented on app/facades/movies_facade.rb:
```diff
@@ -10,12 +10,16 @@ def self.top40
-    [1, 2].reduce([]) do |_memo, page|
-      MovieService.film_finder(page, query)[:results].each do |movie|
-        @movies << Film.new(movie)
+    if query.empty?
+      top40
```
Nice idea to display the Top 40! I think this may be better in the controller where you may have originally put it. One thing maybe we could also do is still show them a flash message like you had it before, that way the user doesn't somehow think the empty search did nothing.

Gvieve committed:
Delete search_for_good_movie_data.yml

Avoid merge conflicts, will record another cassette when we run our test suite again

Gvieve committed:
Delete searched_movie_data.yml

Avoid merge conflicts, will record another cassette when we run our test suite again

Commit:
Delete fixture files

Commit:
Add fixture files

Commit:
Merge branch 'api_testing' of github.com:harrison-blake/viewing_party into api_testing

Commit:
Merge branch 'main' into api_testing

Commit:
Recreate fixture files
